### ISSUE TITLE
Test accuracy not crossing 0.50

### ISSUE THREAD

**Issue by vinayakkailas** [meijieru/crnn.pytorch#153]
HI, I have tried training for my own data. Intially i trained with 150k data. Then 300k and finally i trained with half a million data. But in all cases my test accuracy never crossed 0.50. The model converges in around 5-6 epochs itself with accuracy values between 0.40 and 0.50. I have tried using learning rates 0.01, 0.001,0.0001. No improvements. However all the validation results are showing correct labels.

**Comment by Aurora11111** [comment]
my print is strange:
[24/25][432/432] Loss: 11.319855
Start val
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe6\xbd\xad\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe9\xbe\x88\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe6\xbf\x91\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe6\xba\xa7\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe6\xae\x96\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe7\x97\xa3\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe8\xa2\x88\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe6\xae\xbf\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe5\x87\x80\n'   
b'\\xe5\\\\\\x9-0\xb1\\nn' => b'\xe5\x90\xb1\n'   , gt: b'\xe9\x9e\x8d\n'   
Test loss: 11.234461, accuray: 0.000156

**Comment by Avhirup** [comment]
How long did it take to train? What dataset did you use?

**Comment by vinayakkailas** [comment]
it doesnt take much time for convergence... I used custom dataset

**Comment by Fighting-JJ** [comment]
Have you solved it?
I trained on dataset synth90k  ， and  test accuracy never crossed 0.86. 
I dont know  is there any promotion can be done.

**Comment by Aurora11111** [comment]
I think it beacause of your trianing image size

**Comment by zhengshangdong** [comment]
@Fighting-JJ Hi,bro.I trained on dataset synth90k too.But the loss is always 12.Even I divide the trainsets into 7,000,000 or 800,000 or 5000 pictures,the loss was not change.And the accuray is 0,It could not recognize any number pictures.Did you meet this problem.I use pytorch 1.0 and python 3.6..

**Comment by hiendoan95** [comment]
I have the same problem when training the model with my own Japanese dataset. The max length is 50 characters (based on data requirement). The accuracy never crossed 0.4. Can you give me some advice?
